src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -91,7 +90,7 @@ protected OptionParser buildOptionParser() {
 		parser.accepts( "__separator_5" );
 
 		parser.accepts( "schema_database", "database name for maxwell state (schema and binlog position)").withRequiredArg();
-		parser.accepts( "max_schemas", "how many old schema definitions maxwell should keep around.  default: 5").withOptionalArg();
+		parser.accepts( "max_schemas", "deprecated.").withOptionalArg();
```

Comment:
no, I want to have a release or two with it deprecated so I don't just break everyone's installation.